Components/DB/Dbal/DbalStepsProvider.php

```diff
@@ -291,4 +291,12 @@ protected function getConnection($context)
 
         return $this->doctrine->getConnection($connectionName);
     }
+
+    /**
+     * @param $context
+     */
+    protected function rollback(&$context)
```

Comment:
In this case you don't need to pass the context as reference

Reply:
will remove this. Thanks @campru